### PR TITLE
config.tf: updated version string for 1.6.10 patch release

### PR DESCRIPTION
**Pull request by Quentin-M**

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -64,7 +64,7 @@ variable "tectonic_versions" {
     alertmanager  = "v0.7.1"
     monitoring    = "1.3.1"
     kubernetes    = "1.6.10+tectonic.1"
```
Comment (diegs) [inline config.tf]: `1.6.10+tectonic.2` please

**Comment by diegs** [comment]
Sorry, also `quay.io/coreos/kube-version-operator:v1.6.10-kvo.2` will be the image.

**Comment by Quentin-M** [comment]
Sure, I figured, thanks =P

**Comment by Quentin-M** [comment]
`pr-merge` does not pass on the 1.6.x branches. Please ignore.

**Comment by Quentin-M** [comment]
are we good?

**Comment by Quentin-M** [comment]
test upgrade from 1.6.10-tectonic.1 1.6.10-tectonic.1

**Pull request merged**